iothub/service/src/Exceptions/IotHubServiceException.cs

```diff
@@ -91,6 +91,18 @@ protected IotHubServiceException(SerializationInfo info, StreamingContext contex
         /// <seealso href="https://docs.microsoft.com/rest/api/iothub/common-error-codes"/>.
         public IotHubServiceErrorCode ErrorCode { get; }
 
+        /// <inheritdoc/>
+        public override string ToString()
+        {
+            var sb = new StringBuilder($"Message: {Message}\nErrorCode: {ErrorCode}, IsTransient: {IsTransient}");
```

Comment:
As we have `StatusCode` and `ErrorCode` properties in "IotHubServiceException", I would like to include both of them together. 
```suggestion
            var sb = new StringBuilder($"Message: {Message}\nStatusCode: {StatusCode}, ErrorCode: {ErrorCode}, IsTransient: {IsTransient}");
```